default.yml

```diff
@@ -239,3 +239,9 @@ RSpec/AlignLeftLetBrace:
 
 RSpec/AlignRightLetBrace:
   Enabled: false
+
+RSpec/MultipleMemoizedHelpers:
+  Max: 10
+
+RSpec/NestedGroups:
+  Max: 5
```

Comment:
Hmm... Do we want to set this? I would be of the camp that as long as they are descriptive this should be ok. Juist at thought?

Reply:
The pain I see with this is with a max of 3, you have a `describe` class and a `describe` method, you only get one `context`

Reply:
I might want the context to be `with a user` and then another context `and another user`

Reply:
oh, I think 3 is way small, I like to adjust this to 10 actually.

Reply:
opposed to going higher?

Reply:
@chrishough Not necessarily.. Excess nesting is not so great just visually but I don't feel too strongly about it.  I just don't even understand the motivation for this rule to be honest

Reply:
I am ok with 5, was more curious of the thought process for it.